packages/cloudflare/src/cli/build/build-worker.ts

```diff
@@ -66,6 +69,18 @@ export async function buildWorker(config: Config): Promise<void> {
   patchWranglerDeps(config);
   updateWebpackChunksFile(config);
 
+  await build({
```

Comment:
nit: move this to the patch

Reply:
Shall do. My thinking here was the reason we'd have the patch is because the dynamic import on the cache handler's path doesn't work when you use the next.js config `cacheHandler` option to specify, so there needs to be a patch for initialising the cache, but the building of the cache handler itself behaves as expected and isn't part of a patch to get it to work.